webui/react/src/pages/Admin/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { Tabs } from 'antd';
+import type { TabsProps } from 'antd';
+import React, { useCallback, useMemo, useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import Page from 'components/Page';
+import useFeature from 'hooks/useFeature';
+import usePermissions from 'hooks/usePermissions';
+import { useSettings } from 'hooks/useSettings';
+import GroupManagement from 'pages/Settings/GroupManagement';
+import UserManagement from 'pages/Settings/UserManagement';
+import { paths } from 'routes/utils';
+import { ValueOf } from 'shared/types';
+
+export const TabType = {
+  GroupManagement: 'Group Management',
+  UserManagement: 'User Management',
+} as const;
+
+export type TabType = ValueOf<typeof TabType>;
+
+type Params = {
+  tab?: TabType;
+};
+
+const TAB_KEYS = {
+  [TabType.UserManagement]: 'user-management',
+  [TabType.GroupManagement]: 'group-management',
+};
+const DEFAULT_TAB_KEY = TabType.UserManagement;
+
+const SettingsContent: React.FC = () => {
+  const navigate = useNavigate();
+  const { tab } = useParams<Params>();
+  const [tabKey, setTabKey] = useState<TabType>(tab || DEFAULT_TAB_KEY);
+  const { updateSettings } = useSettings<object>({
```

Comment:
https://github.com/determined-ai/determined/blob/8957c026b344a64b2f868f4fa8b2af7aa89ad5b9/webui/react/src/hooks/useSettings.ts#L23-L27
i think `SettingsConfig` is better than `object`

Reply:
I don't think useSetting is used anyway on this page, removed this part